packages/build/tests/plugins/fixtures/config_mutate_set_undefined/netlify.toml

```diff
@@ -1,2 +1,5 @@
 [[plugins]]
 package = "./plugin.js"
+
+[build]
+command = "test"
```

Comment:
Many tests are modifying `build.command` but were erroneously missing it in the original `netlify.toml` of the fixture. This PR fixes this, which changes some test snapshots.